_posts/05-06-01-Internationalization-and-Localization.md

```diff
@@ -144,13 +145,13 @@ Now that you understood the basis of how plural rules works - and if you didn't,
 on the [LingoHub tutorial](lingohub_plurals) -, you might want to copy the ones you need from a [list][plural] instead
 of writing them by hand.
 
-When calling out Gettext to do localization on sentences with counters, you'll have to give him the
+When calling out Gettext to do localization on sentences with counters, you will have to give him the
 related number as well. Gettext will work out what rule should be in effect and use the correct localized version.
 You will need to include in the `.po` file a different sentence for each plural rule defined.
 
 ### Sample implementation
 After all that theory, let's get a little practical. Here's an excerpt of a `.po` file - don't mind with its format,
-but instead the overall content, you'll learn how to edit it easily later:
+but with the overall content instead; you will learn how to edit it easily later:
```

Comment:
This is an incorrect correction, because the original sentence could use some improvements.

I suggest simply removing all of: `- don't mind with its format, but instead the overall content, you'll learn how to edit it easily later`.   Just leave it as:  `Here's an excerpt of a '.po' file:`